src/radmin_fmt_plug.c

```diff
@@ -172,16 +182,17 @@ static int cmp_exact(char *source, int index)
 
 static void radmin_set_key(char *key, int index)
 {
-	// this code assures that both saved_key[index] gets null-terminated (without buffer overflow)
+	// This code assures that both saved_key[index] gets null-terminated (without buffer overflow)
 	char *cp = &saved_key[index][strnzcpyn(saved_key[index], key, PLAINTEXT_LENGTH + 1)+1];
-	// and is null padded up to 100 bytes.  We simply clean up prior buffer, up to element 99, but that element will never be written to
+	// and is null padded up to 100 bytes. We simply clean up prior buffer,
+	// up to element 99, but that element will never be written to.
 	if (cp < &saved_key[index][99])
 		while (*cp) *cp++ = 0;
 }
 
 static char *get_key(int index)
 {
-	// assured null teminated string.  Just return it.
+	// Assured null teminated string, just return it.
```

Comment:
NUL

Reply:
Ack. Should I use `NULL` or simply `NUL`? I have not seen usage of `NUL` previously, hence the question.